tests/dragonfly/instance.py

```diff
@@ -124,6 +124,12 @@ def __init__(self, params: DflyParams, args):
             if threads > 1:
                 self.args["num_shards"] = threads - 1
 
+        if "disable_serialization_max_chunk_size" not in self.args:
+            # Add 1 byte limit for big values
+            self.args["serialization_max_chunk_size"] = 1
```

Comment:
does it realy makes sense to run always the big value serialization flow in regression tests?
it is good for extensive testing at the beging but then we dont test the other flow in code

Reply:
I wanted this to run for a week or so to make sure that we don't see any other new bugs. I will revert this afterwards. It was Roman's idea back in the summer :smile:

Reply:
point is I either keep this PR and run the reg tests every 3 hours manually or I introduce this change and wait for a week :smile: